### ISSUE TITLE
Is it possible to remove auto /m character in datetime mask

### ISSUE THREAD

**Issue by eduardo-mior** [RobinHerbots/Inputmask#2471]
- Is there an option or property that allows you to remove, or disable, or replace the **m** character that is automatically inserted in the "datetime" mask? 
- OS: **All**
- Browser: **All** 
- Inputmask version: **5.0.5**

Code sample to reproduce the problem:
```javascript
 $ ("#dateInput") .inputmask ("datetime", {
     inputFormat: "dd/mm/yyyy",
     jitMasking: true
 });
```
Screenshot: 
![image](https://user-images.githubusercontent.com/32434418/108385827-ce28d300-71ea-11eb-8a69-2fe15f1bedf0.png)
  
  
**EDIT:**
I used the _onBeforeWrite_ event to remove the automatic **m**, but I don't know if this is correct. Is there a better way to do this?
```javascript
 onBeforeWrite: function (_event, characters) {
     if (characters[3] === "m") {
         characters[3] = ""
     }
 }
```

**Comment by adlerdias** [comment]
Hello,

You can try using a placeholder with a space.

![image](https://user-images.githubusercontent.com/307859/108394031-06ccaa80-71f3-11eb-8ae9-a42caacafce4.png)

`
Inputmask("datetime", {
  inputFormat: "dd/mm/yyyy",
  placeholder: " ",
  /* jitMasking: true */
}).mask('input#date');
`

https://jsfiddle.net/adlersd/wk4goj56/

**Comment by eduardo-mior** [comment]
This works in parts. With this the placeholder is no longer the **m** character and becomes a blank space. I wanted to remove the placeholder completely.
 
![image](https://user-images.githubusercontent.com/32434418/108426099-17dce200-721a-11eb-91d7-2b464bfd85fc.png)

I tried `placeholder: ""` but was unsuccessful.

**Comment by adlerdias** [comment]
To achieve this new specification you need to enable jitMasking

```javascript
Inputmask("datetime", {
  inputFormat: "dd/mm/yyyy",
  placeholder: " ",
  jitMasking: true
}).mask('input#date');
```

**Comment by eduardo-mior** [comment]
> To achieve this new specification you need to enable jitMasking
> 
> ```js
> Inputmask("datetime", {
>   inputFormat: "dd/mm/yyyy",
>   placeholder: " ",
>   jitMasking: true
> }).mask('input#date');
> ```

I'm doing just that, but after typing the second number, it automatically inserts a blank space and a forward slash. I wish you hadn't added that blank space, just the bar.

**Comment by RobinHerbots** [comment]
Duplicate of #2438.

This is fixed int version 5.0.6-beta......

**Comment by eduardo-mior** [comment]
Thanks. Thanks. I'm still using version 5.0.5 (last stable).
